backend/component/dbfactory/dbfactory.go

```diff
@@ -97,7 +97,7 @@ func (d *DBFactory) GetDataSourceDriver(ctx context.Context, instance *store.Ins
 	if err != nil {
 		return nil, err
 	}
-	masterPassword, err := common.Unobfuscate(dataSource.MasterName, d.secret)
+	masterPassword, err := common.Unobfuscate(dataSource.MasterObfuscatedPassword, d.secret)
```

Comment:
Obfuscated or unobfuscated?

Reply:
Yes, I signed CLA, but I haven't add my name to contributors list.
Obfuscated is correct.

Reply:
I already add my name to contributors list. Thanks.